### PR TITLE
Add feature --watch by including chokidar

### PR DESCRIPTION
**Pull request by christiankozalla**
1. Check if the watch flag has been passed
2. Construct an array of expanded file paths in case it is a batched command
3. Load chokidar, pass input paths to its watch method
4. Pass minify(...) as a callback to chokidars onchange method

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -144,7 +145,14 @@ function cli(process, beforeMinifyCallback) {
 
   // ... and do the magic!
   if (program.args.length > 0) {
-    minify(process, options, configurations, expandGlobs(program.args));
+    if (inputOptions.watch) {
+      var inputPaths = expandGlobs(program.args).map(function (path) { return path.expanded; });
```
Comment (jakubpawlowicz) [inline index.js:149]: Since we use `expandGlobs(program.args)` three times here, should we extract it into a variable?

File: package-lock.json [hunk 2]
```diff
@@ -1,14 +1,607 @@
 {
   "name": "clean-css-cli",
   "version": "5.4.2",
-  "lockfileVersion": 1,
```
Comment (jakubpawlowicz) [inline package-lock.json:4]: Can you perhaps regenerate this file with older NPM to minimize number of changes? We may update to lockfile v2 later.

**Review by jakubpawlowicz — CHANGES_REQUESTED** [review]

**Comment by jakubpawlowicz** [comment]
Otherwise looks good!

File: package-lock.json (hunk 2 above)
Comment (christiankozalla) [reply package-lock.json:4]: Alright, I've set the lockfile back to version 1 by using npm v6 in order to install chokidar. Sorry about that, upgrading to v2 was just a side-effect I have not foreseen while I was on npm v7

File: index.js (hunk 1 above)
Comment (christiankozalla) [reply index.js]: I've extracted the call to expandGlobs() into a variable "expandedGlobs"

**Comment by jakubpawlowicz** [comment]
:+1: for changes - I'll pull the PR and give it a try locally before merging. If it's all good then we can release it.

**Comment by jakubpawlowicz** [comment]
All looks good, let's merge it.

**Comment by jakubpawlowicz** [comment]
Two comments for after merge:

- I think we should rebuild all files on first run before 'chokidar' watches files, this is rather a well established approach
- I think we should output to stdout sth like 'File `${path}` has changed. Regenerating optimized file...'

**Comment by jakubpawlowicz** [comment]
Ok, it's done. Please let me know @christiankozalla if you want to add those two simple changes or should I. Thanks!

**Comment by christiankozalla** [comment]
Thank you for testing and merging!

I believe it's better if you could add these two changes, because right now I am not able to do the work, because I lying in bed sick. Nothing severe though

**Comment by jakubpawlowicz** [comment]
Gotcha, get well and many thanks for your contribution!

**Comment by jakubpawlowicz** [comment]
And it's released in clean-css 5.5.0. Thanks!

**Pull request merged**